misc/ballerinalang-data-mapper/build.gradle

```diff
@@ -0,0 +1,112 @@
+apply from: "$rootDir/gradle/javaProjectWithExtBalo.gradle"
+apply from: "$rootDir/gradle/baseNativeStdLibProject.gradle"
+
+//apply plugin: 'jacoco'
+
+configurations {
+    dist {
+        transitive false
+    }
+    dependency {
+        transitive false
+    }
+    libs
+}
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    implementation project(':ballerina-lang')
+    implementation project(':ballerina-parser')
+    implementation project(':ballerina-runtime')
+    implementation project(':language-server:language-server-commons')
+    implementation project(':language-server:language-server-compiler')
+    implementation project(':language-server:language-server-core')
+    implementation project(':language-server:language-server-cli')
+    implementation project(':openapi-ballerina:openapi-validator')
+    implementation project(':openapi-ballerina:openapi-to-ballerina-generator')
+    implementation project(':openapi-ballerina:ballerina-to-openapi-generator')
+
+    // Standard libs
+    implementation project(':ballerina-auth')
```

Comment:
We might not need all of these modules, let's remove them as soon as we merge the PR

Reply:
Okay.